src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -132,7 +132,7 @@ Runtime.GraphQL is null ||
     }
 
     [JsonIgnore]
-    public string DefaultDataSourceName { get; private set; }
+    public string DefaultDataSourceName { get; set; }
```

Comment:
so that we can set it with the Loader, which we need to pass along the same name from one hot-reaload to another. This is because we have some validation that happens at request time and if this changes, that validation fails.